src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -235,6 +235,14 @@ namespace winrt::TerminalApp::implementation
         _showDialogHandlers(*this, FindName(L"AboutDialog").try_as<WUX::Controls::ContentDialog>());
     }
 
+    
+    // Method Description:
+    // - Show a dialog with Shortcuts information. Displays the app's defult keybindings
```

Comment:
```suggestion
    // - Show a dialog with Shortcuts information. Displays the app's default keybindings
```

Reply:
From a grammatical perspective, this would be better:
```suggestion
    // - Show a dialog with shortcut information. Displays the app's default keybindings
```